packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/configurations/ModelConfiguration.java

```diff
@@ -28,6 +29,9 @@ public class ModelConfiguration extends TerariumAsset {
 	@TSOptional
 	private UUID simulationId;
 
+	@TSOptional
+	private Timestamp temporalContextDate;
```

Comment:
```suggestion
	private Timestamp temporalContext;
```

Date means a full date, but with a timestamp you could uses seconds as well. If you see what I mean?

Reply:
Makes sense, done!